src/ApparentHorizons/Strahlkorper.hpp

```diff
@@ -121,6 +121,16 @@ class Strahlkorper {
     return strahlkorper_coefs_;
   }
 
+  /// @{
+  /*!
+   * The physical radius at each collocation point, obtained by
+   * transforming the coefficients to physical space.
+   */
+  SPECTRE_ALWAYS_INLINE DataVector radius() const {
+    return ylm_.spec_to_phys(strahlkorper_coefs_);
+  }
+  /// @}
+
```

Comment:
At some point I think we agreed that any functions on Strahlkorpers should be member functions only if they need to be member functions.  So it seems to me that this `radius()` function should be a free function and not a member function.

Reply:
I decided to go with option 1, so I created StrahlkorperFunctions.hpp and friends for the radius() function